### PR TITLE
adding firebase crashlytics demo project

### PR DESCRIPTION
**Pull request by icarusdust**

### CODE REVIEW

File: integrations/firebase_crashlytics_demo_project/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+This project is a starting point for a Flutter application.
+
+A few resources to get you started if this is your first Flutter project:
+
+- [Lab: Write your first Flutter app](https://flutter.dev/docs/get-started/codelab)
+- [Cookbook: Useful Flutter samples](https://flutter.dev/docs/cookbook)
+
+For help getting started with Flutter, view our
+[online documentation](https://flutter.dev/docs), which offers tutorials,
+samples, guidance on mobile development, and a full API reference.
```
Comment (nevercode-kevin) [inline integrations/firebase_crashlytics_demo_project/README.md:16]: This looks like the generic Flutter README text. Use an example from one of the other demo projects to tell people what this project is, and to look at the codemagic.yaml

**Review by nevercode-kevin — CHANGES_REQUESTED** [review]

File: integrations/firebase_crashlytics_demo_project/README.md (hunk 1 above)
Comment (icarusdust) [reply integrations/firebase_crashlytics_demo_project/README.md]: Done!

File: integrations/firebase_crashlytics_demo_project/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+Or as an alternative it could be added in the **pubspec.yaml**:
+
+```
+dependencies:
+  flutter:
+    sdk: flutter
+  firebase_crashlytics: ^2.5.2
+```
+
+As soon as your build finishes successfully, debug symbols are generated, however if they need them to be present in the **Codemagic UI**, then the following path needs to be configured in **codemagic.yaml** under the artifacts section:
```
Comment (nevercode-kevin) [inline integrations/firebase_crashlytics_demo_project/README.md:21]:
```suggestion
As soon as your build finishes successfully, debug symbols are generated.  However, if you want them to be displayed in the **Codemagic UI** on the build page, then the following path needs to be configured in **codemagic.yaml** under the artifacts section:
```

**Review by nevercode-kevin — CHANGES_REQUESTED** [review]

File: integrations/firebase_crashlytics_demo_project/README.md (hunk 2 above)
Comment (icarusdust) [reply integrations/firebase_crashlytics_demo_project/README.md]: Done!

File: integrations/firebase_crashlytics_demo_project/README.md [hunk 3]
```diff
@@ -18,7 +18,7 @@ dependencies:
   firebase_crashlytics: ^2.5.2
 ```
 
-As soon as your build finishes successfully, debug symbols are generated, however if they need them to be present in the **Codemagic UI**, then the following path needs to be configured in **codemagic.yaml** under the artifacts section:
+As soon as your build finishes successfully, debug symbols are generated.  However, if you want them to be displayed in the **Codemagic UI** on the build page, then the following path needs to be configured in **codemagic.yaml** under the artifacts section:
```
Comment (nevercode-kevin) [inline integrations/firebase_crashlytics_demo_project/README.md:21]:
```suggestion
As soon as your build finishes successfully, debug symbols are generated. However, if you want them to be displayed in the **Codemagic UI** on the build page, then the following path needs to be configured in **codemagic.yaml** under the artifacts section:
```

**Review by nevercode-kevin — CHANGES_REQUESTED** [review]

**Review by nevercode-kevin — APPROVED** [review]
LGTM

**Pull request merged**